CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+- Added `http` namespace with low-level APIs ([#211](https://github.com/opensearch-project/opensearch-ruby/pull/211))
```

Comment:
Let's just say what we added, "Added `http.get`, `post`, ..."? I would remove the second "added guide" line altogether.